package.json

```diff
@@ -2,20 +2,21 @@
   "name": "nextjs-preact-demo",
   "version": "1.0.0",
   "scripts": {
-    "dev": "next",
-    "build": "next build && next export -o build",
+    "dev": "cross-env NODE_ENV=development next",
+    "build:ssg": "cross-env NODE_ENV=production next build && next export -o build",
+    "build:ssr": "cross-env NODE_ENV=production next build",
```

Comment:
I've split these up to make it clear for a user how to differentiate between ssg and ssr